build.gradle

```diff
@@ -246,6 +209,44 @@ switch (DefaultPlatform.RESOURCE_PREFIX) {
 // downloaded libs.
 def libsRootDir = (System.getenv('CI') || ext.libsGitSha == null) ? rootProject.rootDir : rootProject.buildDir
 
+subprojects {
+    apply plugin: 'digital.wup.android-maven-publish'
+
+    // This allows to invoke Gradle like `./gradlew publishToRootProjectBuildDir` (equivalent to
+    // `./gradlew publish`) and also `./gradlew publishToProjectBuildDir`.
+    publishing {
+        repositories {
+            maven {
+                name = "rootProjectBuildDir"
+                url "file://${project.rootProject.buildDir}/maven"
+            }
+            maven {
+                name = "projectBuildDir"
+                url "file://${project.buildDir}/maven"
+            }
+        }
+    }
+
+    apply plugin: NativeLibsPlugin
+    nativeLibs {
+        nss {
+            libRoot = "${libsRootDir}/libs"
```

Comment:
Please file a follow-up to make `libsRoot` part of the `nativeLibs` extension instead of each "lib" block.

Reply:
I tried to do this but do not know how.